src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
A good argument could be made for adding these flags to the top-level swupd command, rather than each sub-command.  1) The end-result is to set global config variables, and 2) these config options apply to every sub-command